inst/@sym/mrdivide.m

```diff
@@ -27,27 +27,7 @@
 
 function z = mrdivide(x, y)
 
-  % Dear hacker from the distant future... maybe you can delete this?
-  if (isa(x, 'symfun') || isa(y, 'symfun'))
-    warning('OctSymPy:sym:arithmetic:workaround42735', ...
-            'worked around octave bug #42735')
-    z = mrdivide(x, y);
-    return
-  end
```

Comment:
i remove it because now we can call directly mrdivide and there is the warning, avoid doube warning